miq_scripts/run.rb

```diff
@@ -1,4 +1,13 @@
-print "Assinging alert profiles to the Enterprise\n"
+puts "Assinging alert profiles to the Enterprise"
 
 MiqAlertSet.find_by(:guid=>"a16fcf51-e2ae-492d-af37-19de881476ad").assign_to_objects(MiqEnterprise.last)
 MiqAlertSet.find_by(:guid=>"ff0fb114-be03-4685-bebb-b6ae8f13d7ad").assign_to_objects(MiqEnterprise.last)
+
+puts "Enabling C&U roles"
+
+server_config = MiqServer.my_server.get_config()
+roles = server_config.config[:server][:role].split(',')
+roles += ['ems_metrics_collector','ems_metrics_coordinator','ems_metrics_processor']
```

Comment:
`+=` always appends.  You may prefer `|=` which does "union" — appends if not there already.

Reply:
Changed to `|=`, that's certainly safer in this case.